common/src/main/java/org/apache/druid/metadata/MetadataStorageConnectorConfig.java

```diff
@@ -78,6 +82,11 @@ public String getPassword()
     return passwordProvider == null ? null : passwordProvider.getPassword();
   }
 
+  public Properties getDbcpProperties()
+  {
+    return dbcpProperties;
```

Comment:
can we do the null check here and return empty Properties object so that null check is not required in all the other places where this gets called?

Reply:
given we don't do null checks on any of the other members, does it make sense?

Reply:
I see your point ... I thought it would be cleaner to not return null. but it is one of those "preference" things so I will leave it up to you... feel free to skip this comment.

Reply:
I would prefer to do it, but then I feel conflicted since the rest of the members aren't null checked and then don't want to break convention already set.